modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -0,0 +1,589 @@
+
+    if (module->isCameraInSolarSystem()) {
+        module->lookAtTarget(id);
+    }
+
+    return 0;
+}
+
+int setOpacityOfImageLayer(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 3, "lua::setOpacityOfImageLayer");
+    auto [id, i, opacity] = ghoul::lua::values<std::string, int, double>(L);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    TargetBrowserPair* found = module->getPair(id);
+    if (found) {
+        found->setImageOpacity(i, opacity);
+    }
+
+    return 0;
+}
+
+int centerTargetOnScreen(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::centerTargetOnScreen");
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    TargetBrowserPair* pair = module->getPair(id);
+    if (pair) {
+        pair->centerTargetOnScreen();
+    }
+        
+    return 0;
+}
+
+int setSelectedBrowser(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::setSelectedBrowser");
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+        
+    module->setSelectedBrowser(id);
+        
+    return 0;
+}
+
+int createTargetBrowserPair(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 0, "lua::createTargetBrowserPair");
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    int noOfPairs = module->nPairs();
+    std::string nameBrowser = "Sky Browser " + std::to_string(noOfPairs);
+    std::string nameTarget = "Sky Target " + std::to_string(noOfPairs);
+    std::string idBrowser = "SkyBrowser" + std::to_string(noOfPairs);
+    std::string idTarget = "SkyTarget" + std::to_string(noOfPairs);
+    glm::vec3 positionBrowser = { -1.0f, -0.5f, -2.1f };
+    std::string guiPath = "/SkyBrowser";
+    std::string url = "https://data.openspaceproject.com/dist/skybrowser/page/";
+
+    const std::string browser = "{"
+        "Identifier = '" + idBrowser + "',"
+        "Type = 'ScreenSpaceSkyBrowser',"
+        "Name = '" + nameBrowser + "',"
+        "Url = '" + url + "',"
+        "FaceCamera = false,"
+        "CartesianPosition = " + ghoul::to_string(positionBrowser) +
+        "}";
+    const std::string target = "{"
+        "Identifier = '" + idTarget + "',"
+        "Type = 'ScreenSpaceSkyTarget',"
+        "Name = '" + nameTarget + "',"
+        "FaceCamera = false,"
+        "}";
+
+    global::scriptEngine->queueScript(
+        "openspace.addScreenSpaceRenderable(" + browser + ");",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+
+    global::scriptEngine->queueScript(
+        "openspace.addScreenSpaceRenderable(" + target + ");",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+
+    global::scriptEngine->queueScript(
+        "openspace.skybrowser.addPairToSkyBrowserModule('" + idTarget + "','"
```

Comment:
Maybe use fmt to format these strings? Potentially makes it a little easier to see what's going on
```
fmt::format("openspace.skybrowser.addPairToSkyBrowserModule({}, {});", idTarget, idBrowser)
```